packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -432,7 +442,10 @@ export class PipelineEditor extends React.Component<
 
         showDialog({
           title: 'Run pipeline',
-          body: new PipelineSubmissionDialog({ runtimes: response.runtimes }),
+          body: new PipelineSubmissionDialog({
+            exporting,
```

Comment:
for readability I'd recommend `exporting: exporting` even if it's redundant since the other value has to be written that way